packages/stark-core/src/modules/error-handler/actions/stark-error-handler.actions.ts

```diff
@@ -0,0 +1,28 @@
+import { Action } from "@ngrx/store";
+
```

Comment:
We should remove the `stark-` prefix from the name of this file... it is already implicit since it is part of the Stark packages ;)   We should just keep the prefix in the name of the class